src/main/java/org/folio/dataexp/service/export/strategies/HoldingsExportStrategy.java

```diff
@@ -83,10 +113,29 @@ public Optional<String> getIdentifierMessage(UUID id) {
     return Optional.empty();
   }
 
-  protected List<JSONObject> getHoldingsWithInstanceAndItems(Set<UUID> holdingsIds, GeneratedMarcResult result, MappingProfile mappingProfile) {
+  @Override
+  public Map<UUID,MarcFields> getAdditionalMarcFieldsByExternalId(List<MarcRecordEntity> marcRecords, MappingProfile mappingProfile) {
+    return new HashMap<>();
+  }
+
+  protected List<JSONObject> getHoldingsWithInstanceAndItems(Set<UUID> holdingsIds, GeneratedMarcResult result,
+      MappingProfile mappingProfile, ExportRequest exportRequest, boolean lastSlice, boolean lastExport) {
```

Comment:
Hi @obozhko-folio , could please a bit rework usage of boolean lastSlice, boolean lastExport arguments as not clear the usage of this flags.